src/components/NavBar/NavBar.js

```diff
@@ -16,8 +19,27 @@ class NavBar extends Component {
     </Tooltip>
   );
 
+  showCurrentPursuance = (pursuances) => {
+    let id = parseInt(window.location.pathname.slice(-1), 10);
+    let rawPursuance = pursuances[id];
+    if (rawPursuance !== undefined) {
+      return rawPursuance.name;
+    } else {
+      return "Jump to pursuance";
+    }
+  }
+
+  handleJumpToPursuance = (e) => {
+    window.location.replace(`/pursuance/${e}`);
```

Comment:
@thevaleriemack ReactRouter is useful for routing here, like this: https://github.com/PursuanceProject/pursuance/blob/9c59ec0ec1b0fe0f3c7dd3bdbd63c0b213c9a477/src/components/Content/TaskHierarchy/Task/Task.js#L49-L54

Reply:
history is also coming up undefined

Reply:
I think the component's `connect` call may need to be wrapped using `withRouter` to get `history` to show up. Let's check...

Reply:
`withRouter` will cause both `history` and `match` to be populated (yaaay) -- https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md